apps/emqx/src/emqx_connection.erl

```diff
@@ -552,13 +552,13 @@ handle_msg({quic, Data, _Stream, #{len := Len}}, State) when is_binary(Data) ->
     inc_counter(incoming_bytes, Len),
     ok = emqx_metrics:inc('bytes.received', Len),
     when_bytes_in(Len, Data, State);
-handle_msg(check_cache, #state{limiter_buffer = Cache} = State) ->
-    case queue:peek(Cache) of
+handle_msg(check_limiter_buffer, #state{limiter_buffer = Buffer} = State) ->
+    case queue:peek(Buffer) of
         empty ->
-            activate_socket(State);
+            handle_info(activate_socket, State);
```

Comment:
should it follow L928?

Reply:
this covers L928.

Reply:
did the same as 
https://github.com/emqx/emqx/blob/d99f033d7491836f1f62b8704ecb8fcf6661d302/apps/emqx/src/emqx_connection.erl#L596

Reply:
ok, I have `handle_info({sock_error, Reason}, State)` on my branch. maybe it is lagging.

Reply:
https://github.com/emqx/emqx/blob/d99f033d7491836f1f62b8704ecb8fcf6661d302/apps/emqx/src/emqx_connection.erl#L928

Reply:
I don't like the idea of calling handle_info when handling `check_limiter_buffer`, it confused reader.

Reply:
I agree, it confused me too.